src/window/window_wrapper/mod.rs

```diff
@@ -166,52 +150,49 @@ impl GlutinWindowWrapper {
 
     pub fn draw_frame(&mut self, dt: f32) {
         let window = self.windowed_context.window();
-        let scaling = window.scale_factor();
-        let new_size = window.inner_size();
+        let scale_factor = 1.0; // window.scale_factor();
```

Comment:
Is the scale factor here supposed to be commented out?

Reply:
That's temporary. I'll remove it alltogether and use PhysicalSize.